src/native-common/AccessibilityUtil.ts

```diff
@@ -54,6 +60,28 @@ const componentTypeMap = {
 };
 
 export class AccessibilityUtil extends CommonAccessibilityUtil {
+    // Native platform specific instance for AccessibilityUtil. 
+    private _instance: typeof AndroidAccessibilityUtil | typeof iOSAccessibilityUtil; 
```

Comment:
These should both be the same type. You should define an interface (or abstract base class) that both iOS and Android need to implmenet